ankihub/gui/reviewer.py

```diff
@@ -62,20 +64,30 @@ def _setup_webview(self):
             )
 
         self.splitter = aqt.QSplitter()
+        self.container = aqt.QWidget()
+        container_layout = aqt.QVBoxLayout()
+        self.container.setLayout(container_layout)
 
         # Create a QWebEngineProfile with persistent storage
         profile = aqt.QWebEngineProfile("AnkiHubProfile", parent_widget)
         profile.setHttpUserAgent(
             "Mozilla/5.0 (Windows NT 10.0; Win64; x64) Chrome/91.0.4472.124 Safari/537.36"
         )
 
-        # Create the main web view
+        # Create the web views
         self.webview = aqt.webview.AnkiWebView()
         self.webview.setPage(CustomWebPage(profile, self.webview._onBridgeCmd))
         self.webview.setUrl(aqt.QUrl(self.urls_list[0]["url"]))
-
-        self.webview.hide()
+        self.header_webview = aqt.webview.AnkiWebView()
+        self.header_webview.setSizePolicy(
+            aqt.QSizePolicy(
+                aqt.QSizePolicy.Policy.Expanding, aqt.QSizePolicy.Policy.Fixed
+            )
+        )
```

Comment:
:wrench: There is something wrong with the height of the header webview. For me it looks like this: 
<img src="https://github.com/user-attachments/assets/c88fce44-257e-4b76-bf54-7f46d608fd95" width="300" />

I added a `self.header_webview.setFixedHeight(1)` call and then it looked much better:
<img src="https://github.com/user-attachments/assets/d085a9e0-e0cb-4934-bc15-047c3298098a" width="300" />

However, I also noticed that there is a margin between the two webviews, we should try to remove this. We could do this in a different PR if that's better.
It looks like there is also a margin on the left and the right side of the sidebar, which we might also want to remove.

Reply:
> I added a self.header_webview.setFixedHeight(1) call and then it looked much better:

Feel free to add it to the appropriate place. I couldn't reproduce the issue on Windows.

Reply:
I added an `adjustSize` instead and this fixed it for me.

Unfortunately there is still space between the webviews. Also, it looks like the header webview is slightly less wide than the other webview. 
<img src="https://github.com/user-attachments/assets/b54e8ce4-adc2-45a0-95c9-3debd0419e9a" width="350" />